assets/src/js/admin/onboarding-wizard/utils/index.js

```diff
@@ -29,84 +38,62 @@ export const redirectToSetupPage = () => {
  * @since 2.8.0
  */
 export const saveSettingWithOnboardingAPI = ( setting, value ) => {
-	// Example shape of returned data
-	return {
-		setting,
-		value,
-	};
-
 	// Logic for connecting to the Onboarding API
-	// An object with action: 'save' and satting: ${value} is passed to the API
+	// An object with action: 'save' and setting: ${value} is passed to the API
 	// An object of the same shape should be returned to confirm the value was stored as expected
```

Comment:
The use of a nested dispatched action doesn't allow for returning a confirmation. Is this methodology important for the current architecture? Or should we consider a "Redux way" of handling API errors?

Reply:
Hmm...I think we will want to be able to handle confirmation, more so for implementing loading states in a future issue. Since this kind of thing can be a can of worms, I'm gonna suggest moving forward with merge the way you have implemented things, but with the understanding that I'll open an issue to handle loading states that may result in some rethinking of this logic. Because the onboarding wizard app is small, and doesn't need to be tremendously modular, we have some greater flexibility about this kind of thing.